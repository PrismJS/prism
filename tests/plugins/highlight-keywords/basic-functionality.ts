import { createTestSuite } from '../../helper/prism-dom-util';


describe('Highlight Keywords', () => {
	const { it } = createTestSuite({
		languages: 'javascript',
		plugins: 'highlight-keywords'
	});

	it('should highlight keywords', ({ util }) => {
		util.assert.highlightElement({
			language: 'javascript',
			code: `import * from ''; const foo;`
		});
	});

});
