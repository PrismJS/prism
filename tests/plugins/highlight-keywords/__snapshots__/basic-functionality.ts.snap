// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Highlight Keywords should highlight keywords 1`] = `
"<span class=\\"token keyword module keyword-import\\">import</span>
<span class=\\"token operator\\">*</span>
<span class=\\"token keyword module keyword-from\\">from</span>
<span class=\\"token string\\">''</span>
<span class=\\"token punctuation\\">;</span>
<span class=\\"token keyword keyword-const\\">const</span>
foo
<span class=\\"token punctuation\\">;</span>
"
`;
